static/js/src/advantage/subscribe/react/APICalls/usePostCustomerInfoForPurchasePreview.jsx

```diff
@@ -22,28 +22,24 @@ const usePostCustomerInfoAnon = () => {
       state: country === "US" ? usState : caProvince,
     };
 
-    if (VATNumber) {
```

Comment:
I had to remove this check in order for the data to be updated on the back-end (even if no VAT number supplied) so that the subsequent preview call returns the correct price (for example when switching countries). This fixes another issue as well: https://github.com/canonical-web-and-design/commercial-squad/issues/174

Reply:
I believe this is behaving correctly now:

![Kapture 2021-08-24 at 08 17 26](https://user-images.githubusercontent.com/7452681/130579511-3c96e63c-8526-42da-b14d-22c176e7b6a5.gif)